### ISSUE TITLE
{Freeway-Bug}[SampleCard-StockUpdate; Config-microsoft-teams-light.json]: The color contrast ratio for the text related to 'Rise in Stock' in green is having 2.07:1 is not sufficient.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8470]
### Target Platforms

Android

### SDK Version

2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment:
Test Environment
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.03.31.1

Pre-requisite:
Import-Sample Cards->WeatherCompact.json; Config-microsoft-teams-light.json

Repro Steps:
1. Navigate to the above sample card.
2. Observe the contrast issue for the text related to 'Rise in Stock' in green

Actual Result:
The color contrast ratio for the text related to 'Rise in Stock' in green is having 2.07:1 is not sufficient.

Expected Result:
The color contrast ratio for the text should be at least 4.5:1.

Note:
This issue is repro for below configs,
skype.json (CCA is 3.73:1)

User Experience:
Low vision users will get impact and will not able to tell the text if its color contrast ratio is not sufficient against the background

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

![MAS1 4 3The color contrast ratio for the text related to 'Rise in Stock' in green is having 2 071 is not sufficient](https://user-images.githubusercontent.com/89897257/232712004-00f747be-49a5-4fce-b2a7-dfa8ca43b900.jpg)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro hence closing this issue.

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/d55e2728-8757-40fb-8b9c-ca4cb138a9de)
